README.rst

```diff
@@ -196,7 +196,7 @@ Related projects
 * `gmtmex <https://github.com/GenericMappingTools/gmtmex>`__: A Matlab/Octave wrapper
   for GMT.
 
```

Comment:
Question: Do we want to mention other similar projects in this section? E.g. `cartopy`?

And if we do mention `cartopy`, would it be ok to put Liam's new ROSES 2021 video here (c.f. #613) since he does do a bit of a PyGMT/Cartopy comparison?

Reply:
I would prefer to not list other projects that don't share the connection with GMT because it would be quite subjective what gets included.